src/main/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitChecker.java

```diff
@@ -72,7 +72,7 @@ public void checkApiRateLimit(@NonNull TaskListener listener, GitHub github) thr
                         )));
                     }
                     listener.getLogger().println(GitHubConsoleNote.create(System.currentTimeMillis(),
-                            "Jenkins is using \"Throttle for Normalize\" to try and evenly distribute GitHub API requests. Use \"Throttle on Over\" to have Jenkins restrict GitHub API requests only when near or above the rate limit."));
+                            "Jenkins is attempting to evenly distribute GitHub API requests. To configure a different rate limiting strategy, such as having Jenkins attempt to evenly distribute GitHub API requests, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings."));
```

Comment:
```suggestion
                            "Jenkins is attempting to evenly distribute GitHub API requests. To configure a different rate limiting strategy, such as having Jenkins restrict GitHub API requests only when near or above the GitHub rate limit, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings."));
```

Reply:
Good catch, updated in 04b3a2feabe4159c454cfa113b59402d120bc360